policy/src/main/java/brooklyn/enricher/TimeWeightedDeltaEnricher.java

```diff
@@ -41,6 +42,8 @@
  * presented as a units/timeUnit based on the event timing.
  * <p>
  * NB for time (e.g. "total milliseconds consumed") use {@link TimeFractionDeltaEnricher}
+ * <p>
+ * See also {@link YamlTimeWeightedDeltaEnricher} designed for use from YAML.
```

Comment:
I've marked the older classes `@Beta` and added a reference.  I think they should go in time, but not sure yet...